src/parse.ts

```diff
@@ -189,10 +204,25 @@ const parseRelationship = (variableName: string, originalObject: string): Formul
     })
 }
 
+const parseInstance = (variableName: string): FormulaIdentifierInfo[] => {
+  if ((variableName.match(/\./g) || []).length > 2) {
+    return []
+  }
+  if (getObject(variableName).endsWith(RELATIONSHIP_SUFFIX)) {
+    return []
+  }
+  return [{
+    type: 'instance',
```

Comment:
Doesn't this require complimentary support in the SF adapter?

Reply:
Yup. But we need to publish this package first.